src/main/java/lotto/domain/policy/SixMatch.java

```diff
@@ -0,0 +1,19 @@
+package lotto.domain.policy;
+
+import lotto.domain.lotto.Lotto;
+import lotto.domain.winpolicy.Policy;
+
+public class SixMatch extends Policy {
+  public SixMatch() {
+    super(2000000000);
```

Comment:
```suggestion
    super(2_000_000_000);
```
가독성을 높여줄 수 있는 방식이 있답니다 :)

Reply:
숫자형 데이터를 보기좋게하는 방법을 몰랐었는데  
알려주셔서 감사합니다!!